ansible-scylla-node/tasks/ssl.yml

```diff
@@ -1,20 +1,33 @@
 ---
 - name: generate a self-signed CA and use it for node certs
   block:
+    - name: Check CA folder exists
```

Comment:
This is not necessary, as the ansible file module can do that by default in the next task.

Reply:
You're right, I've removed this check